pkg/tuple/relationship.go

```diff
@@ -0,0 +1,26 @@
+package tuple
+
+import (
+	v1 "github.com/authzed/authzed-go/proto/authzed/api/v1"
+)
+
+// StringObjectRef marshals a *v1.ObjectReference into a string.
+func StringObjectRef(ref *v1.ObjectReference) string {
+	return ref.ObjectType + ":" + ref.ObjectId
+}
+
+// StringSubjectRef marshals a *v1.SubjectReference into a string.
+func StringSubjectRef(ref *v1.SubjectReference) string {
+	if ref.OptionalRelation != "" {
```

Comment:
Why the change from builder here?

Reply:
It's _faster_ than `fmt.Sprintf`, that's how I got it 85% down.

Reply:
Right, but is it faster than the `string.Builder`?

Reply:
yep!